### ISSUE TITLE
EPIC: Full support for WCAG AA 2.1 Compliance

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#1114]
We had the IDRC assess us against WCAG 2.1 AA standards, and we received a "Supports" rating for all but 2 criteria. We received a "Partially Supports" for 2 criteria, and I've created Github issues to address them:

- [x] 2.5.3 Label in Name: #1127
- [x] 1.4.13 Content on Hover or Focus: #1128

**Comment by klemay** [comment]
NOTE: [one of our Summer 2020 pilots](https://hypothes-is.slack.com/archives/C8TPC8XMK/p1590704888106200) will need concrete plans for getting to 2.1 in order to continue using Hypothesis in the fall.

**Comment by klemay** [comment]
We had the IDRC assess us against WCAG 2.1 AA standards, and we received a "Supports" rating for all but 2 criteria. We received a "Partially Supports" for 2 criteria, and I've created Github issues to address them:

2.5.3 Label in Name: https://github.com/hypothesis/product-backlog/issues/1127 
1.4.13 Content on Hover or Focus: https://github.com/hypothesis/product-backlog/issues/1128

**Comment by chrisshaw** [comment]
We [achieved this milestone](https://web.hypothes.is/blog/everyone-included-hypothesis-accessibility-reaches-wcag-2-1/) in our last audit.
